### ISSUE TITLE
fq.gz is not always geneExpression

### ISSUE THREAD

**Issue by allaway** [nf-osi/nfportalutils#135]
This will cause some errors - this could also be WGS, WES, etc. 
https://github.com/nf-osi/nfportalutils/blob/58e8d84af34b99e59f5b4748a6257c53cbb3f807/R/annotation_reference.R#L63

**Comment by anngvu** [comment]
Yeah, I think the refinement was to try to include assay context if possible. These are not bulletproof yet.
